src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        int figureNumber = 11;

```

Comment:
According to the checklist, all magic numbers in your code should be constants. Consider defining a constant for the number of figures to be generated.